crates/ruff/src/rules/flake8_bugbear/snapshots/ruff__rules__flake8_bugbear__tests__B029_B029.py.snap

```diff
@@ -0,0 +1,15 @@
+---
+source: crates/ruff/src/rules/flake8_bugbear/mod.rs
+expression: diagnostics
+---
+- kind:
+    ExceptWithEmptyTuple: ~
+  location:
+    row: 7
+    column: 0
+  end_location:
+    row: 8
+    column: 18
+  fix: ~
+  parent: ~
+
```

Comment:
Do you have `insta` installed? (`cargo install cargo-insta`) You should be able to run `cargo test --lib`, then `cargo insta review` to review and check in the generated snapshot.

Reply:
@charliermarsh  yeah it's weird I did run `cargo test --all` and `cargo insta review` but the file was not updated 🤔

Reply:
![image](https://user-images.githubusercontent.com/43336371/220194253-fa1c3572-f031-438a-a89b-cf8f29e2e9ee.png)

Reply:
Oh, you need to add the test to `crates/ruff/src/rules/flake8_bugbear/mod.rs` (hopefully the pattern is clear once you open up that file).

Reply:
@charliermarsh nvm 🤦 I accidentally deleted it upon resolving the merge conflict here ![image](https://user-images.githubusercontent.com/43336371/220196193-0375b715-b36d-4092-8fa0-8fc2231b75b3.png)

will restore it asap

Reply:
Happens to the best of us :D